spec/fixtures/responses/queries/freaks/freaks.json

```diff
@@ -17,10 +17,6 @@
               "lastName": "doe",
               "description": "a simple freak",
               "email": "freak@gmail.com",
-              "photo": {
```

Comment:
Why delete the `photo`s here?

Reply:
![image](https://user-images.githubusercontent.com/24190790/130762020-f2317d00-cbab-421e-9023-2b8ab74f4aa3.png)